libs/vm-modules/tests/unit/ml/ml_model_tests.cpp

```diff
@@ -848,4 +853,18 @@ TEST_F(VMModelTests, model_with_accuracy_metric)
   EXPECT_LE(metrics->elements.at(1), 1);
 }
 
+TEST_F(VMModelTests, DISABLED_model_sequential_flatten)
+{
```

Comment:
Why disabled? Would it fail now?

Reply:
yes - that is because the estimator will set maximum charge, which will cost too much for the VM to execute. That is expected behaviour for now, but see https://fetchai.atlassian.net/browse/ML-377